src/sql/workbench/common/editor/query/queryEditorInput.ts

```diff
@@ -196,6 +204,11 @@ export abstract class QueryEditorInput extends EditorInput implements IConnectab
 		}
 	}
 
+	protected async notifyConnectionUriChanged(newUri: string): Promise<void> {
```

Comment:
This is better named as `changeConnectionUri` since it's actually doing that action